### PR TITLE
fix created leveldb dir name

### PR DESCRIPTION
**Pull request by r4mmer**
### Acceptance Criteria

Some tests merged after we changed how we name the leveldb store used the old naming system.
This only affects some tests but when running they will leave a directory with the xpub name instead of creating inside the `testdata.leveldb` like other tests.

- Use the correct parameters when instantiating the leveldb store on tests

### Security Checklist
- [ ] Make sure you do not include new dependencies in the project unless strictly necessary and do not include dev-dependencies as production ones. More dependencies increase the possibility of one of them being hijacked and affecting us.

### CODE REVIEW

File: __tests__/storage/common_store.test.js [hunk 1]
```diff
@@ -177,7 +177,8 @@ describe('registered tokens', () => {
 
   it('should work with leveldb store', async () => {
     const xpriv = new HDPrivateKey();
-    const store = new LevelDBStore(DATA_DIR, xpriv.xpubkey);
+    const walletId = crypto.Hash.sha256(Buffer.from(xpriv.xpubkey)).toString('hex');
```
Comment (pedroferreira1) [inline __tests__/storage/common_store.test.js:180]: Can we wait the other PR and use the util created here?

Comment (tuliomir) [reply __tests__/storage/common_store.test.js:180]: If we do, I think it's important to also refactor the other instances of similar code in the tests ( about 14, on my latest search )

**Review by tuliomir — APPROVED** [review]

File: __tests__/utils/utxo.test.js [hunk 2]
```diff
@@ -1,13 +1,14 @@
 /**
  * Copyright (c) Hathor Labs and its affiliates.
- *
+ *wallet
```
Comment (pedroferreira1) [inline __tests__/utils/utxo.test.js:3]: Remove?

**Review by pedroferreira1 — APPROVED** [review]

File: __tests__/utils/utxo.test.js (hunk 2 above)
Comment (r4mmer) [reply __tests__/utils/utxo.test.js]: Done

**Pull request merged**